config/initializers/sidekiq.rb

```diff
@@ -1,3 +1,5 @@
+Sidekiq.strict_args!(false)
```

Comment:
maybe we would bump Sidekiq :)

Reply:
I haven't used Sidekiq before so just solved the exception, open to hear more for this